Doc/whatsnew/3.10.rst

```diff
@@ -388,6 +388,12 @@ Optimizations
   for more details. (Contributed by Victor Stinner and Pablo Galindo in
   :issue:`38980`)
 
+* The constant slices (e.g ``[1:], [1:], a[::2]``) are now constructed at the
+  compile time and stored in the code objects. Slicing with them are now faster
+  and depending on the type, can increase the speed between %5-%30.
+  (Contributed by Batuhan Taskaya in :issue:`42454`)
```

Comment:
```suggestion
* Constant slices (e.g ``[1:], [1:], a[::2]``) are now constructed at compile time
  and stored in code objects. Slicing operations involving them are now between
  %5-%30 faster. (Contributed by Batuhan Taskaya in :issue:`42454`)
```

Reply:
Resolved in e68c8cd89393afa4fa40f74a6e9a3c414bbd104a (line wrapped)